cmd/root.go

```diff
@@ -17,7 +17,10 @@ var (
 	// The usage examples and help strings should reflect
 	// the actual name of the binary.
 	BinaryName string
-	Out        io.Writer
+	// Out is used by the download command
```

Comment:
`Out` should actually be used in multiple commands. It looks like we have `out` and `Out` defined in the cmd package. If #483 is good by you, we should merge those changes in then rebase your branch onto master.